spaghetti/util.py

```diff
@@ -206,21 +211,21 @@ def snapPointsOnSegments(points, segments):
     node2segs = {}
 
     for segment in segments:
-        head,tail = segment.vertices
-        x0,y0 = head
-        x1,y1 = tail
-        if (x0,y0) not in node2segs:
-            node2segs[(x0,y0)] = []
-        if (x1,y1) not in node2segs:
-            node2segs[(x1,y1)] = []
-        node2segs[(x0,y0)].append(segment)
-        node2segs[(x1,y1)].append(segment)
-        x0,y0,x1,y1 =  segment.bounding_box
+        head, tail = segment.vertices
+        x0, y0 = head
+        x1, y1 = tail
+        if (x0, y0) not in node2segs:
+            node2segs[(x0, y0)] = []
+        if (x1, y1) not in node2segs:
+            node2segs[(x1, y1)] = []
+        node2segs[(x0, y0)].append(segment)
+        node2segs[(x1, y1)].append(segment)
+        x0, y0, x1, y1 = segment.bounding_box
         x0 -= SMALL
```

Comment:
woah, I think this should use the smallest numpy epsilon float, not an arbitrarily large number, which may cause problems if the x/y ranges are small.

Reply:
btw this is not critical to solve to merge; this would go in a separate bug & get fixed there!

Reply:
@ljwolf I just added some spaces in there. The actual code was original `pysal.network`.

Reply:
yeah, no I know it's not something added here, hence why it's cool to not have to solve it for this merge :smile: